keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/binary/ArcTan2Vertex.java

```diff
@@ -10,16 +12,31 @@
 
 public class ArcTan2Vertex extends DoubleBinaryOpVertex {
 
+    private static final String X_NAME = LEFT_NAME;
+    private static final String Y_NAME = RIGHT_NAME;
+
     /**
      * Calculates the signed angle, in radians, between the positive x-axis and a ray to the point (x, y) from the origin
      *
      * @param x x coordinate
      * @param y y coordinate
      */
-    public ArcTan2Vertex(DoubleVertex x, DoubleVertex y) {
+    @ExportVertexToPythonBindings
+    public ArcTan2Vertex(@LoadParentVertex(X_NAME) DoubleVertex x,
+                         @LoadParentVertex(Y_NAME) DoubleVertex y) {
         super(x, y);
     }
 
+    // No SaveParentVertex annotation needed as DoubleBinaryOpVertex.getLeft() will be used when saving the vertex
+    public DoubleVertex getX() {
```

Comment:
Seems a bit tedious having to bang in these getters for `left` and `right` when it goes to the super anyway. I'm a bit confused on why we need the getters and the `X_NAME` and `Y_NAME` - can't we just use `LEFT` and `RIGHT` from super?

Reply:
Similar to how you've got no getters for the `unary` ops

Reply:
Seems like we've got this for a few of the vertices already - like `MaxVertex`

Reply:
Aha - this is actually Jon's code.  I'll have a scan through and make it consistent where we only have getters on the parent I think.